src/index.js

```diff
@@ -323,8 +321,9 @@ class ServerlessIOpipePlugin {
   createFile() {
     const debug = createDebugger('createFile');
     debug('Creating file');
-    const iopipeInclude = `const iopipe = require('iopipe')({token: '${this.getOptions()
-      .token}'});`;
+    let { token = process.env.IOPIPE_TOKEN } = this.getOptions();
+    token = token ? `'${token}'` : 'process.env.IOPIPE_TOKEN';
+    const iopipeInclude = `const iopipe = require('iopipe')({token: ${token}});`;
```

Comment:
@coreylight So I like the option of being able to set the `IOPIPE_TOKEN` from the deployment point. But if the user uses serverless's `environment` setting in their `serverless.yml` I don't think this will catch that. Would it also make sense to have a check for `IOPIPE_TOKEN` in the handler wrapper at invocation time?

Reply:
The intention of this is to set the token to a value if one is found in the custom section, or on the `process.env` object. If neither is found, it falls back to the `process.env.IOPIPE_TOKEN` variable that could be populated through the lambda env vars in the aws console, for example. I believe this gives the user the most flexibility?

Reply:
The process.env.IOPIPE_TOKEN is passed through by reference, not by value, so it would work at runtime @kolanos

Reply:
However, could we add a test for this case to be sure @coreylight?

Reply:
I don't think that's the issue. The scenario I think will be confusing for the developer is as follows:

1. Developer sets `export IOPIPE_TOKEN=....` locally on their dev machine.
2. They create a `serverless.yml` with an `environment` section specifying an `IOPIPE_TOKEN`.
3. They deploy their function from their dev machine.

Now assuming the `IOPIPE_TOKEN` is different in 1 and 2, which one shoul win out? In this case, 1 would be used, but I think the dev would want 2 to be used in the deployed function as that was what they explicitly set in their `serverless.yml`.

Reply:
You have to specify in serverless.yml the behavior of your env var (where it comes from). So it would be up to the dev if they wanted to pull from their local env or use a default or whatever. Right?